### PR TITLE
:heavy_plus_sign: feat: Save user's theme choice

### PR DESCRIPTION
**Pull request by mendoc**
Now the app saves the user's choice of theme to local storage. Even after refreshing the page, the theme is maintained.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -15,7 +15,7 @@
             <div class="params-theme">
                 <span>Switch theme !</span>
                 <div>
-                    <input type="checkbox" id="toggle" class="checkbox" onclick="changeTheme(checked);" />  
+                    <input type="checkbox" id="toggle" class="checkbox" onclick="changeTheme(this);" />
```
Comment (gouri-panda) [inline index.html]: what is `this` in the field

Comment (mendoc) [reply index.html]: This is no longer useful. Sorry.

**Review by gouri-panda — APPROVED** [review]

**Comment by mendoc** [comment]
@gouri-panda I pulled dev branch but my work doesn't appear. Some explanation ?

**Comment by gouri-panda** [comment]
@mendoc what do you mean your work doesn't appear? And where?

**Comment by mendoc** [comment]
My work doesn't appear in the current dev branch version. I added save user's theme choice feature. I tried in local and my code doesn't appear in app.js.

I created a function named checkTheme() in app.js file.

**Comment by mendoc** [comment]
I solve the problem. Thank's for your attention.

**Pull request merged**